src/ApplicationModule.js

```diff
@@ -177,9 +175,7 @@ const __class = declare('crm.ApplicationModule', [ApplicationModule], {
       },
     }));
 
-    this.registerView(new CalendarMonthView());
```

Comment:
Keep the view registered. However, go into the view's properties and set expose to false. This way it won't show up in the configure menu anymore.